src/components/AccountDetails/index.tsx

```diff
@@ -16,7 +16,7 @@ import { ExternalLink, LinkStyledButton, ThemedText } from '../../theme'
 import { shortenAddress } from 'utils'
 import { ExplorerDataType, getExplorerLink } from 'utils/getExplorerLink'
 import { ButtonSecondary } from '../Button'
-import StatusIcon from '../Identicon/StatusIcon'
+import StatusIcon from '@cow/modules/wallet/api/components/StatusIcon'
```

Comment:
Is it intentional to modify original uni files?

Reply:
yes, I even deleted some. I was tempted to delete some modals we don't use, but I think this should be a clean up PR done separately, so instead i just fixed some imports

Reply:
account details will probably need to be the next PR